homeassistant/components/sense/binary_sensor.py

```diff
@@ -111,6 +141,17 @@ def should_poll(self):
         """Return the deviceshould not poll for updates."""
         return False
 
+    @property
+    def device_state_attributes(self):
+        """Return the device specific state attributes."""
+        realtime_data = self._data.get_realtime()
+        devices = realtime_data.get("devices", {})
+        for device in devices:
+            if device["id"] == self._id:
+                return {ATTR_WATTS: round(device["w"], 1)}
```

Comment:
This seems like it should be a sensor.

Reply:
Yes it probably should have been.  I didn't change it over due to the increase in scope and it would make it a breaking change.

Reply:
Probably a good followup PR

Reply:
We shouldn't add it to binary sensor in this PR.

Reply:
We shouldn't add it to binary sensor in this PR.

Reply:
Will strip it out.  I can likely take out the initial update_realtime as well (need to verify)

Reply:
Looks like we can strip out update_realtime.